inc/utype.h

```diff
@@ -145,6 +147,35 @@ extern struct arabicforms {
     unsigned int required_lig_with_alef: 1;
 } ArabicForms[256];	/* for chars 0x600-0x6ff, subtract 0x600 to use array */
 
```

Comment:
I'm not talking about ArabicForms. I'm talking about the following function declarations. I think that, since we are using local declarations, it is worthwhile adding a comment noting where they are defined. It's not a big deal, but it's easy to do when making other adjustments.